com.unity.netcode.gameobjects/Samples/ClientNetworkTransform/Scripts/InputNetworkTransform.cs

```diff
@@ -0,0 +1,36 @@
+using Unity.Netcode.Components;
+using UnityEngine;
+
+namespace Unity.Netcode.Samples
+{
+    // server driven network transform with client side input sending
+    public class InputNetworkTransform : NetworkTransform
```

Comment:
I don't think we should have something like this. This can be done via composition. The implementation here is also very flawed and non trivial to get right. We will show users how to pass input to a NetworkTransform in our goldenpath etc. There is no need to have this in a separate sample.

Reply:
I agree, if it were just up to me, this whole class wouldn't exist.

Reply:
*this whole class + this ServerRPC

Reply:
removed this whole class. whether we keep the delta RPC is still TBD